core/src/components/UnifiedSearch/SearchableList.vue

```diff
@@ -46,7 +46,8 @@
 						:wide="true"
 						@click="itemSelected(element)">
 						<template #icon>
-							<NcAvatar :user="element.user" :show-user-status="false" :hide-favorite="false" />
+							<NcAvatar v-if="element.isUser" :user="element.user" :show-user-status="false" :hide-favorite="false" />
```

Comment:
```suggestion
							<NcAvatar v-if="element.isUser" :user="element.user" :show-user-status="false" />
```
Don't think this prop exists 🙈